mi/dataset/driver/pco2a_a/sample/pco2a_a_sample_recovered_driver.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+"""
+@package mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_recovered_driver
+@file mi/dataset/driver/pco2a_a/sample/pco2a_a_sample_recovered_driver.py
+@author Sung Ahn
+@brief Recovered driver for pco2a_a_sample data parser.
+
+"""
+
+from mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_driver import process, \
+    RECOVERED_PARTICLE_CLASSES
+from mi.core.versioning import version
+
+
+@version("15.6.2")
```

Comment:
If this is a new version, it should be 0.1.0 or something like that. Looks like you copied this from another driver.

Reply:
I dropped the recovered,telemetered drivers, so moved the parse function into the main driver and modified the version to 0.1.0 there.